### ISSUE TITLE
Small bug in discriminator.py

### ISSUE THREAD

**Issue by dhruvpatel108** [shaohua0116/WGAN-GP-TensorFlow#3]
Seems like there is a small bug [here ](https://github.com/shaohua0116/WGAN-GP-TensorFlow/blob/5b1e895604c48e006269e5068da0ec72f5c75b11/discriminator.py#L26). conv2d_res doesn't take norm as an argument. So, you might wanna remove it.

**Comment by PaulZhangIsing** [comment]
Yes, as long as we comment the norm line, it is able to work smoothly
